frontend/src/styles/Caption.css

```diff
@@ -83,21 +85,38 @@
     border: solid 1px #f44b42;
 }
 
+.caption-container .caption-content h1 a{
+  font-family: 'Teko';
+  font-size: 20px;
+}
+
 .caption-content {
     margin: auto 0;
 }
 
+.caption-container h1 {
```

Comment:
Everywhere caption-container is defined, caption-container-accepted and caption-container-rejected should also have the same style